--- a/config/Config.qml
+++ b/config/Config.qml
@@ -36,1 +36,4 @@
-    property string configDir: (Quickshell.env("XDG_CONFIG_HOME") || (Quickshell.env("HOME") + "/.config")) + "/ambxst/config"
+    readonly property string configRoot: Quickshell.env("AMBXST_CONFIG_ROOT") || ((Quickshell.env("XDG_CONFIG_HOME") || (Quickshell.env("HOME") + "/.config")) + "/ambxst")
+    property string configDir: configRoot + "/config"
+    property string keybindsPath: configRoot + "/binds.json"
--- a/shell.qml
+++ b/shell.qml
@@ -177,4 +177,6 @@
     CompositorConfig {
         id: compositorConfig
     }
+
+    CompositorKeybinds {}
